README.md

```diff
@@ -57,12 +56,17 @@ This will ensure the queue that is generated is compatible with the expectations
 
 ### Execution
 
-Execute the worker to pick up messages in the queue and process them into the database:
+Execute the worker to pick up messages in the queue and process them into the
+database:
 
-    bundle exec ./script/worker 1 start
+    ```bundle exec ./bin/amqp_client start```
 
```

Comment:
The script was a layer of abstraction added for the old deployment project, and isn't used in production any more. It just complicates things, and can hide useful error messages.